utils/obsproc/util.h

```diff
@@ -221,33 +221,32 @@ namespace gdasapp {
           oops::Log::test() << checksum(datetime_, "datetime") << std::endl;
         }
 
-       // Convert JulianDate and second
-       void JulianDateTime(const std::string & str, int64_t & DAwindow) {
+       // Convert ISO 8601 formatted datetime to epoch seconds
+       void DateToSeconds(const std::string & str, int64_t & start_end_seconds) {
```

Comment:
the camel casing doesn't follow the proper convention, it should be `dateToSeconds` for a function. Also, sorry for being picky, but `start_end_seconds` for your variable name is confusing.